app/javascript/components/visualization/StudyViolinPlot.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect } from 'react'
```

Comment:
Per our chat: I updated this module in #829, and will plan to merge my updates over there.